README.md

```diff
@@ -200,7 +203,9 @@ tasks.withType<Test> {
 }
 
 dependencies {
-  testImplementation("io.kotest:kotest-runner-junit5-jvm:<version>")
+  testImplementation("io.kotest:kotest-runner-junit5-jvm:<version>") // for kotest framework
+  testImplementation("io.kotest:kotest-assertions-core:<version>") // for kotest core assertions
+  testImplementation("io.kotest:kotest-assertions-jvm:<version>") // for kotest jvm assertions
```

Comment:
We can simplify this to 

 testImplementation("io.kotest:kotest-runner-junit5-jvm:<version>") // for kotest framework
  testImplementation("io.kotest:kotest-assertions-core-jvm:<version>") // for kotest core jvm assertions

Reply:
Done